examples/iam_config.json

```diff
@@ -17,6 +17,9 @@
     "read_write": [
       "test_bucket_read_write/*",
       "test_bucket_read_only/write_folder/*"
+    ],
+    "deny": [
+      "test_bucket_read_write/sensitive_table"
```

Comment:
does the deny path also need to end `/*` to deny everything in that path? I don't know the answer but would seem logical

Reply:
yeah I guess so, updated